OpenRA.Platforms.Default/Shader.cs

```diff
@@ -25,11 +24,9 @@ class Shader : ThreadAffine, IShader
 
 		protected int CompileShaderObject(ShaderType type, string name)
 		{
-			string ext = type == ShaderType.VertexShader ? "vert" : "frag";
-			string filename = "glsl{0}{1}.{2}".F(Path.DirectorySeparatorChar, name, ext);
-			string code;
-			using (var file = new StreamReader(GlobalFileSystem.Open(filename)))
-				code = file.ReadToEnd();
+			var ext = type == ShaderType.VertexShader ? "vert" : "frag";
+			var filename = "glsl{0}{1}.{2}".F(Path.DirectorySeparatorChar, name, ext);
+			var code = File.ReadAllText(filename);
```

Comment:
You can throw in one spot and catch in another to present useful information.
This was my point.